index.php

```diff
@@ -63,19 +87,37 @@
   if (isset($_POST["entry_1922375458"])) { //organisations
     $requested_orgs = filter_var_array($_POST["entry_1922375458"], FILTER_SANITIZE_STRING);
     foreach ($requested_orgs as $requested_org) {
-      if (in_array($requested_org, $allowed_orgs) && !empty($requested_org) ) { //!!!!FIX ME!!!!
+      if (in_array($requested_org, $allowed_orgs) && !empty($requested_org) ) { //!!!!FIX ME!!!!I would but I don't know how you're broken good sir...
```

Comment:
A bunch of these FIX MEs were introduced in the very first commit:
https://github.com/IATI/IATI-Query-Builder/commit/530c4c51#diff-828e0013b8f3bc1bb22b4f57172b019dR35

In all cases, the `$allowed` arrays (the whitelists) were empty, and the `in_array` conditions were flipped. So I guess the FIX MEs referred to that.

It looks like this has since been fixed (e.g. `$allowed_orgs` is now populated, and is actually used as a whitelist), and while all the other FIX MEs were removed, this one wasn’t.

In short: I think it’s safe to remove.

Reply:
Wizard